docs/Using Any API/make-a-http-get-request.md

```diff
@@ -48,14 +48,14 @@ contract APIConsumer is ChainlinkClient {
     
     /**
      * Network: Kovan
-     * Oracle: 0x2f90A6D021db21e1B2A077c5a37B3C7E75D15b7e
-     * Job ID: 29fa9aa13bf1468788b7cc4a500a45b8
+     * Oracle: 0xc57B33452b4F7BB189bB5AfaE9cc4aBa1f7a4FD8
+     * Job ID: d5270d1c311941d0b08bead21fea7747
      * Fee: 0.1 LINK
      */
     constructor() public {
         setPublicChainlinkToken();
-        oracle = 0x2f90A6D021db21e1B2A077c5a37B3C7E75D15b7e;
-        jobId = "29fa9aa13bf1468788b7cc4a500a45b8";
+        oracle = 0xc57B33452b4F7BB189bB5AfaE9cc4aBa1f7a4FD8;
```

Comment:
can we add (Chainlink devrel node) in the comments here so we know where this node is coming from?

Reply:
Added it in the comments above. Let me know how it looks, thanks!